shpc/main/modules/__init__.py

```diff
@@ -104,17 +104,15 @@ def _uninstall(self, module_dir, name, force=False):
         """
         Sub function, so we can pass more than one folder from uninstall
         """
-        if os.path.exists(module_dir) and not force:
-            msg = "%s, and all content below it? " % name
-            if utils.confirm_uninstall(msg, force):
-                self._cleanup(module_dir)
-                logger.info("%s and all subdirectories been removed." % name)
-
-        elif os.path.exists(module_dir) and force:
+        if os.path.exists(module_dir):
+            if not force:
+                msg = "%s, and all content below it? " % name
+                if not utils.confirm_uninstall(msg, force):
+                    return
             shutil.rmtree(module_dir)
             logger.info("%s and all subdirectories have been removed." % name)
         else:
-            logger.info("%s does not exist." % name)
+            logger.warning("%s does not exist." % name)
```

Comment:
This works! Could you confirm you see the message in mustard yellow when it doesn’t exist? It makes me suspicious why I chose info instead of warning!

Reply:
<img width="404" alt="Screenshot 2022-03-16 at 02 44 03" src="https://user-images.githubusercontent.com/623458/158506317-3df43c49-aa4d-4a0a-b8da-629b0f964a9b.png">

Reply:
Perfect thank you.